### PR TITLE
Updated readme and contributing

### PR DESCRIPTION
**Pull request by jessica-tw**
Signed-off-by: jessica-tw <jessica.olinda@zup.com.br>

- Updated README and CONTRIBUTING.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,103 @@
...
+You can suggest a change, a fix, or improvements to our documentation, for example: 
+- Add missing information.
+- Fix a grammar or code error.
+- Suggest a new section.
+- Translate. 
+
+###  **Contributing Guide**
+Check out Beagle's [**contributing guide**](https://github.com/ZupIT/beagle/blob/main/CONTRIBUTING.md). 
+
+### **Open an issue**
```
Comment (hectorcustodiozup) [inline README.md:81]: Just a point of attention here. Opening an issue does not mean that the same person has to fix and start a pull request. Those are two different steps that could be tangled or not.

**Review by hectorcustodiozup — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jessica-tw) [reply README.md:81]: I see, maybe change the way it is there, for example:
- You can use use the 'Edit this page' in the documentation to make a fix or open a new issue. See below how you can do that: 

What do you suggest?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,28 +1,114 @@
...
+
+## **How to contribute?** 
+
+You can suggest a change, a fix, or improvements to our documentation, such as: 
+- Add missing information.
+- Fix a grammar or code error.
+- Suggest a new section.
+- Translate. 
+
+### **Open an issue**
```
Comment (hectorcustodiozup) [inline CONTRIBUTING.md:75]:
```suggestion
### **Opening issues and pull requests**
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,28 +1,114 @@
...
+Follow the steps below: 
+
+**Step 1.** Access [**GitHub Issues**](https://github.com/ZupIT/beagle-docs/issues) and open an issue;
+
+**Step 2.** Describe the issue or change you want to make; 
+
```
Comment (hectorcustodiozup) [inline CONTRIBUTING.md:81]:
```suggestion
You can also suggest solutions for opened issues following steps 3 to 5
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,28 +1,114 @@
...
+**Step 3.** Make your changes;
```
Comment (hectorcustodiozup) [inline CONTRIBUTING.md:82]:
```suggestion
**Step 3.** Commit your changes to a new branch and upload it to the repository;
```

**Review by hectorcustodiozup — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hectorcustodiozup) [reply README.md:81]: I added some suggestions, but again those are just extra details. Your explanation is just fine. Feel free to merge the changes as they are.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,28 +1,114 @@
...
+
+**Step 4.** Open a pull request open a PR to the published branch, i. e., **`release-VERSION`** and fill in the template;
```
Comment (hectorcustodiozup) [inline CONTRIBUTING.md:84]:
```suggestion
**Step 4.** Open a pull request to the published branch, i. e., **`release-VERSION`** and fill in the template; 
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,28 +1,114 @@
...
+
+**Step 1.** Access [**GitHub Issues**](https://github.com/ZupIT/beagle-docs/issues) and open an issue;
+
+**Step 2.** Describe the issue or change you want to make; 
+
+**Step 3.** Make your changes;
+
+**Step 4.** Open a pull request open a PR to the published branch, i. e., **`release-VERSION`** and fill in the template; 
+
+**Step 5.** In GitHub, send a pull request to **`beagle:main`**;
```
Comment (hectorcustodiozup) [inline CONTRIBUTING.md:86]:
```suggestion
**Step 5.** Also open a pull request to the default branch **`beagle:main`**;
```

File: README.md (hunk 1 above)
Comment (jessica-tw) [reply README.md:81]: I liked your ideas! Thanks a lot ;)

**Review by hectorcustodiozup — APPROVED** [review]

**Pull request merged**